Content.Shared/Traits/Prototypes/TraitPrototype.cs

```diff
@@ -87,4 +87,22 @@ public sealed partial class TraitPrototype : IPrototype
     /// </summary>
     [DataField]
     public List<ProtoId<MoodEffectPrototype>>? MoodEffects { get; private set; } = default!;
+
+    /// <summary>
+    ///     The list of all Factions that this trait removes.
+    /// </summary>
+    /// <remarks>
+    ///     I can't actually Validate these because the proto lives in Shared.
```

Comment:
Does ProtoId not work? `customTypeSerializer: typeof(PrototypeIdListSerializer<NpcFactionPrototype>)` should work in DataField

Reply:
No because the NpcFactionPrototype literally doesn't exist in Shared lol. It's an ancient system that exists solely in Server.

Reply:
Oh, I thought you meant NpcFactionPrototype was in shared, should clarify that